README.md

```diff
@@ -82,6 +82,20 @@ try {
 }
 ```
 
+## Strict mode
+
+If you want to use a more strict method (without normalizing TIN number, that is, the raw TIN number) use strict parameter in check or isValid functions this way (See https://github.com/loophp/tin/pull/39):
+
+```
+TIN::fromSlug('be7110.2512345')->check(true);
+```
+
+```
+$bool = TIN::fromSlug('be7110.2512345')->isValid(true);
+```
+In this case, both examples will not validate raw TIN number.
```

Comment:
```suggestion
## Strict Mode

If you want to use a stricter method (without normalizing the TIN number, that is, using the raw TIN number), use the `strict` parameter in the `check` or `isValid` functions as shown below (Introduced in [#39](https://github.com/loophp/tin/pull/39)). By default, it is set to `false`.

```php
TIN::fromSlug('be7110.2512345')->check(); // Not raw
TIN::fromSlug('be7110.2512345')->check(strict: false); // Not raw
TIN::fromSlug('be7110.2512345')->check(true); // Raw
TIN::fromSlug('be7110.2512345')->check(strict: true); // Raw
```
```